src/workerd/api/worker-loader-test.js

```diff
@@ -510,7 +510,21 @@ export let isolateUniqueness = {
     assert.strictEqual(await anonymousAgain.increment(), 2);
     assert.strictEqual(await anonymousAgain.increment(), 3);
 
-    assert.strictEqual(loadCount, 5);
+    // Test that null/undefined name parameter creates unique isolates each time
+    // Each call should generate a unique name internally.
+    let noName1 = env.loader.get(null, loadCodeCallback).getEntrypoint();
```

Comment:
Feel free to ignore this, but what about allowing `env.loader.get(loadCodeCallback).getEntrypoint()`?